### ISSUE TITLE
[Magus] Summon Flamer Tooltip

### ISSUE THREAD

**Issue by magusgamer69** [WarEmu/WarBugs#16462]
The tooltip for this ability incorrectly states that... "While within 25 feet of your minion you will be empowered by..." However unlike the other 2 minions this was changed to double that (seemingly since the magus/engineer summon reworks some years ago).

There also seems to be a weird deadzone around 40-50 feet where stacks are neither generated or lost unlike the other minions  which there is a clear dropoff. Not sure if this is intended either.

**Comment by Kaelang** [comment]
Please could you submit screenshots of both the tooltip, and the distance to where you're still receiving the buff?

**Comment by magusgamer69** [comment]
![1](https://user-images.githubusercontent.com/64134614/79983379-51fc1c80-846d-11ea-8f5c-4f8dbdb2e1c8.PNG)

**Comment by magusgamer69** [comment]
To reiterate again, I think what happened here is these changes happened long before client control, and when the tooltips were finally updated when it was possible a mistake was made and all the tooltips read the same for distance.

**Comment by cimator** [comment]
Yes, turrets/demons were changed couple years ago, iirc the bombardment turret should give stacks up to 40ft, and between 40-65ft it doesnt give stacks, but you dont lose stacks either, after 65ft you lose stacks. Looks like there is more missing tooltip aswell, like the damage buff going up to %40 and not %20, will let you know if I actually find the patch notes

**Comment by cimator** [comment]
There it is! the main patch note; https://www.returnofreckoning.com/forum/viewtopic.php?f=42&t=18792 and 2 more patch notes with couple adjustments to the turrets/demons;
https://www.returnofreckoning.com/forum/viewtopic.php?f=42&t=18803
https://www.returnofreckoning.com/forum/viewtopic.php?f=42&t=18817
This change was long before the client control so the tooltips couldnt be changed.

**Comment by dalen** [comment]
hmm, this one is a bit tricky to fix, the tooltip for those daemons/turrets are already at the character limit for an ability tooltip, so can't really squeeze more into in there.
